.pre-commit-config.yaml

```diff
@@ -8,7 +8,7 @@ ci:
 
 repos:
 - repo: https://github.com/pre-commit/pre-commit-hooks
-  rev: v4.5.0
+  rev: v4.6.0
```

Comment:
These ase some useful quality assurance checks that make sure files have the right structure, dependencies are up to date, etc.